languages/example/irc.lang.json

```diff
@@ -56,6 +58,8 @@
   "Cannot send verification code there": "Cannot send verification code there",
   "Channel %[1]s has %[2]d persistent modes set": "Channel %[1]s has %[2]d persistent modes set",
   "Channel %s is now unregistered": "Channel %s is now unregistered",
+  "Channel %s list already contains %s": "Channel %s list already contains %s",
+  "Channel %s list does not contain %s": "Channel %s list does not contain %s",
```

Comment:
so on these two strings, the translator may want to swap these variables around. we should make the source strings specify `%[1]s` and `%[2]s` for each param, e.g.:
```
"Channel %[1]s list already contains %[2]s"
"Channel %[1]s list does not contain %[2]s"
```
this can be done in this PR, just make sure to regen the translation files afterwards